include/sys/videoio.h

```diff
@@ -899,8 +899,8 @@ struct v4l2_frmivalenum
 
 struct v4l2_pix_format
 {
-  uint16_t  width;              /* Image width in pixels */
-  uint16_t  height;             /* Image height in pixels */
+  uint32_t  width;              /* Image width in pixels */
+  uint32_t  height;             /* Image height in pixels */
```

Comment:
Do we need more thank 64K x 64K pixels?

Reply:
this change make v4l2_pix_format has the same header compared with v4l2_pix_format_mplane, since some application assume both share the same common header.